### PR TITLE
feat: argo workflows package

### PR DESCRIPTION
**Pull request by omerlh**
Hope PRs are welcomed, amazing project and I needed it for argo workflow so I figured if I'm going to generate it I can open a PR...

### CODE REVIEW

File: .changeset/fast-steaks-love.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@kubernetes-models/argo-workflows": major
```
Comment (tommy351) [inline .changeset/fast-steaks-love.md:2]: The first release should be a minor release.

```suggestion
"@kubernetes-models/argo-workflows": minor
```

File: third-party/argo-workflows/CHANGELOG.md
Comment (tommy351) [inline third-party/argo-workflows/CHANGELOG.md:1]: Please remove this file.

File: third-party/argo-workflows/__tests__/class.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { describe, it, expect, beforeEach } from "vitest";
+import { Workflow } from "../gen/argoproj.io/v1alpha1/Workflow";
+
+describe("Rollout", () => {
```
Comment (tommy351) [inline third-party/argo-workflows/__tests__/class.ts:4]:
```suggestion
describe("Workflow", () => {
```

File: third-party/argo-workflows/README.md
Comment (tommy351) [inline third-party/argo-workflows/README.md:1]: Please update the readme.

Comment (tommy351) [inline third-party/argo-workflows/README.md:1]: Please update this readme for Argo Workflows.

Comment (omerlh) [reply third-party/argo-workflows/README.md:1]: fixed

**Pull request merged**